File: RS2/JamFix.Services/Database/Prijava.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace JamFix.Services.Database
{
    public class Prijava
    {
        public int PrijavaId { get; set; }
        public int KonkursId { get; set; }
        public string Ime{ get; set; }
        public string Prezime { get; set; }
        public string Email { get; set; }
        public string BrojTelefona { get; set; }
        public DateTime DatumPrijave { get; set; }
        public virtual Konkurs Konkurs { get; set; } = null!;
    }
}

File: RS2/UI/jamfix_mobilna/jamfix_mobilna/lib/models/prijava.dart
import 'package:json_annotation/json_annotation.dart';

part 'prijava.g.dart';

@JsonSerializable()
class Prijava {
  int? prijavaId;
  int? konkursId;
  String? ime;
  String? prezime;
  String? email;
  String? brojTelefona;
  DateTime? datumPrijave;

  Prijava(
      {this.prijavaId,
      this.konkursId,
      this.ime,
      this.prezime,
      this.email,
      this.brojTelefona,
      this.datumPrijave});

  factory Prijava.fromJson(Map<String, dynamic> json) =>
      _$PrijavaFromJson(json);

  Map<String, dynamic> toJson() => _$PrijavaToJson(this);
}
